src/main/java/code401challenges/BinarySearch.java

```diff
@@ -0,0 +1,25 @@
+package code401challenges;
+
+public class BinarySearch {
+    public static void main(String[]args){
+
+    }
+

```

Comment:
If you used any sources, make sure you are crediting them here, or in your readme